### ISSUE TITLE
App Request: Chibisafe

### ISSUE THREAD

**Issue by cherryroots** [saltyorg/Sandbox#371]
# Chibisafe

**What does it do? Why? Please describe.**

I'd like to have this included as the other services included in the sandbox either are too heavy(nextcloud) or not maintained anymore (Jirafeau). The following is the opening quote of the github page

> Chibisafe is a file uploader service written in typescript that just works. It's easy to use, easy to deploy, free and open source. It accepts files, photos, documents, anything you imagine and gives you back a shareable link for you to send to others.
>
>You can run it in public mode, user accounts mode or invite-only mode. Big files are automatically chunked to minimize network failure and users can create an API key to use the service [programatically](https://chibisafe.app/docs) and hook it to other things.
>
>The service also comes with a dashboard where you can edit almost every configuration of the instance directly from the UI without having to touch environment or configuration files manually. Control the name, the rate limit, max file size, accepted extensions, meta descriptions, etc directly from our intuitive panel.

## Links to further information

- [x] Link to project homepage or github | [homepage](https://chibisafe.app/) | [github](https://github.com/chibisafe/chibisafe)
- [x] Link to documentation | [Guides](https://chibisafe.app/guides)
- [x] Link to installation instructions |  [Docker compose and traefik](https://github.com/chibisafe/chibisafe/blob/master/packages/next/src/content/guides/running-with-docker-compose-and-traefik.mdx)
- [x] Link to compose file | see doc link

## What are you willing to do to help?

- [ ] I will you help with making docs if this gets picked up.
- [x] I will you help with testing.
- [ ] Other things:

**Comment by owine** [comment]
So this needs two underlying containers, Caddy serving the webui and then Saltbox's Traefik would be on top of that?

**Comment by saltydk** [comment]
You don't need caddy, just multiple routers on the Traefik side to handle the 2 unique paths.

**Comment by owine** [comment]
> You don't need caddy, just multiple routers on the Traefik side to handle the 2 unique paths.

Strange that their sample Traefik deployment still deploys Caddy in between.

**Comment by saltydk** [comment]
Because they don't know what they are doing I suppose.

**Comment by saltydk** [comment]
I mean they have this bit in their NPM guide:
> As Docker by default does not run as your user, this may not be the best location to put Chibisafe in. The more "appropriate" location would be on /srv, so change the place where Chibisafe is installed accordingly if you want to follow best practices.

**Comment by saltydk** [comment]
The only time I've needed another proxy as a go between is when using WSGI. There are probably other edge cases but it is the only one I could think of.

**Comment by cherryroots** [comment]
Yeah I don't think Caddy is required at all as the NPM guide doesn't use it, just a bit odd to have it with Traefik but I guess they don't know how to set up routers
